src/hallway/feed.clj

```diff
@@ -0,0 +1,29 @@
+(ns hallway.feed
```

Comment:
I renamed this namespace since we're technically not using RSS (we're using Atom). My fault obviously, as #4 even includes RSS in the title ;). I tend to use the terms interchangeably since I think RSS is has better recognition... but in code it's probably worth getting the technical details right.